corehq/motech/openmrs/exceptions.py

```diff
@@ -1,12 +1,26 @@
 from __future__ import unicode_literals
 
 
-class OpenmrsConfigurationError(Exception):
+class OpenmrsException(Exception):
+    """
+    It's not us, it's them
+    """
+    pass
```

Comment:
nit: you don't need `pass` if you have a docstring

Reply:
True, but somehow I've always done it. I imagined it was convention, but that could just have been in my head. Leaving for now, but will add a cleanup to my to-do list because I've done it everywhere, forever. :grimacing: